src/SayMore/Model/ArchivingHelper.cs

```diff
@@ -139,6 +166,27 @@ internal static void SetIMDIMetadataToArchive(IIMDIArchivable element, Archiving
 			}
 		}
 
+		internal static void SetRAMPMetadataToArchive(IIMDIArchivable element, ArchivingDlgViewModel model)
+		{
+			var project = element as Project;
+			if (project != null)
+			{
+				foreach (var session in project.GetAllSessions())
+					AddRAMPSession(session, model as RampArchivingDlgViewModel);
+			}
+			else
+			{
+				AddRAMPSession((Session)element, model as RampArchivingDlgViewModel);
+			}
+		}
+
+		internal static void AddRAMPSession(Session saymoreSession, RampArchivingDlgViewModel model)
+		{
+			var contributions = saymoreSession.MetaDataFile.GetValue("contributions", null) as ContributionCollection;
+			if (contributions != null && contributions.Count > 0)
+				model.SetContributors(new ContributionCollection());
```

Comment:
Why is this passing in an empty ContributionCollection? Do we even need this or does it get handled later with a call to SetAdditionalMetsData?